backend/api/fixtures/setup_data.json

```diff
@@ -90,5 +90,87 @@
           "min_age": 11,
           "max_age": null
       }
+  },
+  {
+    "model": "api.eventtype",
+    "pk": 1,
+    "fields": {
+        "distance": 50,
+        "stroke": "M",
```

Comment:
Could you change this one to Fly? please.
A 50 Medley is not posible and also we don't have any fly event.